src/deep-security/lib/Token.js

```diff
@@ -6,6 +6,8 @@
 
 import AWS from 'aws-sdk';
 import {AuthException} from './Exception/AuthException';
+import {CredentialsManager} from './CredentialsManager';
+import {IdentityProvider} from './IdentityProvider';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['IdentityProvider' is defined but never used.](https://www.codacy.com/app/MitocGroup/deep-framework/file/1363209943/issues/source?bid=2895475&fileBranchId=2973909#l10)